docker-compose-sunbird/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: start
+
+start:
+	bash setup_vault.sh docker-compose.yml vault
+	docker compose up -d
+
+destroy:
+	docker compose down
+	sudo rm -rf data db-data
+	docker volume rm demo-mosip-rc_vault-data
```

Comment:
looks like make file does very minimal things,  possible to avoid this dependency ?

Reply:
removed the make file